pom.xml

```diff
@@ -26,6 +26,7 @@
     <folio-spring-base.version>2.0.0</folio-spring-base.version>
     <openapi-generator.version>5.2.0</openapi-generator.version>
     <mapstruct.version>1.4.2.Final</mapstruct.version>
+    <log4j2.version>2.16.0</log4j2.version> <!-- if less then vulnerable to CVE-2021-44228 and CVE-2021-45046-->
```

Comment:
If folio-spring-base with upgraded log4j2 has been released, it's better to use it instead of explicitly specifying log4j version

Reply:
The `parent` for `mod-password-validator` is `spring-boot-starter-parent` and `folio-spring-base` is just a dependency.
Maven properties are not inherited from dependencies, so upgrading folio-spring-base won't upgrade log4j2 dependency version.